monitoring/dashboard.py

```diff
@@ -525,6 +559,167 @@ def top10_errors_by_bucket(title, code):
+    unit=UNITS.OPS_PER_SEC,
+    targets=[
+        Target(
+            expr='sum(rate(s3_cloudserver_quota_evaluation_duration_seconds_count{namespace="${namespace}"}[$__rate_interval])) by(action)',  # noqa: E501
+            legendFormat="{{action}}",
+        )
+    ]
+)
+
+
+averageQuotaLatencies = TimeSeries(
+    title="Average quota evaluation latencies",
+    dataSource="${DS_PROMETHEUS}",
+    lineInterpolation="smooth",
+    spanNulls=3*60*1000,
+    legendDisplayMode="table",
+    legendPlacement="right",
+    legendValues=["min", "mean", "max"],
+    unit=UNITS.SECONDS,
+    targets=[
+        average_quota_latency_target(title="Overall"),
+        average_quota_latency_target(
+            title="Data write (bucket, success)",
+            type='"bucket"', code='~"2.."'),
+        average_quota_latency_target(
+            title="Data write (bucket, exceeded)",
+            type='"bucket"', code='"429"'),
+        average_quota_latency_target(
+            title="Data write (account, success)",
+            type='"account"', code='~"2.."'),
+        average_quota_latency_target(
+            title="Data write (account, exceeded)",
+            type='"account"', code='"429"'),
+        average_quota_latency_target(
+            title="Data write (account & bucket, success)",
+            type='"bucketAccount"',
+            code='~"2.."'),
+        average_quota_latency_target(
+            title="Data write (account & bucket, exceeded)",
+            type='"bucketAccount"',
+            code='"429"'),
+        average_quota_latency_target(
+            title="Data deletion", type='"delete"'),
+    ],
+)
+
+
+averageMetricsRetrievalLatencies = TimeSeries(
+    title="Average utilization metrics retrieval latencies",
+    dataSource="${DS_PROMETHEUS}",
+    lineInterpolation="smooth",
+    spanNulls=3*60*1000,
+    unit=UNITS.SECONDS,
+    targets=[
+        average_quota_retrieval_latency(title="Bucket (success)",
+                                        _class='"bucket"', code='~"2.."'),
+        average_quota_retrieval_latency(title="Account (success)",
+                                        _class='"account"', code='~"2.."'),
+        average_quota_retrieval_latency(title="Bucket (error)",
+                                        _class='"bucket"', code='~"4..|5.."'),
+        average_quota_retrieval_latency(title="Account (error)",
+                                        _class='"account"', code='~"4..|5.."'),
+    ],
+)
+
+
+bucketQuotaCounter = Stat(
+    title="Buckets with quota",
+    description=(
+        "Number of S3 buckets with quota enabled in the cluster.\n"
+        "This value is computed asynchronously, and update "
+        "may be delayed up to 1h."
+    ),
+    dataSource="${DS_PROMETHEUS}",
+    colorMode="value",
+    format=UNITS.SHORT,
+    noValue="0",
+    reduceCalc="lastNotNull",
+    targets=[Target(
+        expr='sum(s3_cloudserver_buckets_quota_count{namespace="${namespace}"})',  # noqa: E501
```

Comment:
max() may be more appropriate than sum, it case we have multiple series at the same time (e.g. pod update maybe?)